src/App.js

```diff
@@ -38,18 +53,30 @@ const App = () => {
     <div className="App antialiased text-gray-900">
       <Navbar></Navbar>
       <div className="container">
-        <form onSubmit={performSearch} className="search-form">
-          <input
-            className="search-bar"
-            type="text"
-            value={search}
-            onChange={updateSearch}
-          />
-          <button className="search-button" type="submit">
-            Submit
-          </button>
-          <RecipeList recipes={recipes} />
-        </form>
+        {isViewDetailsDisplayed ? (
```

Comment:
If you end up making more routes here an easy way to manage them is just through an object and use the keys for your state.

```js
const ROUTE_MAP = {
  HOME: RecipeList,
  DETAILS: DetailView 
}

....
....

const [route, setRoute] = useState("HOME");

const View = ROUTE_MAP[route];

return (
  <View {...viewProps} />
)

```

Reply:
Cool, thanks I will keep this in mind for when I add more routes :)